examples/enforce-gatekeeper/imperative/README.md

```diff
@@ -0,0 +1,88 @@
+# enforce-gatekeeper: Imperative Example
+
+### Overview
+
+This example demonstrates how to imperatively invoke the [enforce-gatekeeper]
+function to validate resources using gatekeeper constraints.
+
+There are 3 resources: a ConstraintTemplate, a K8sBannedConfigMapKeysV1 and a
+ConfigMap.
+The constraint disallows using `private_key` as a key in the ConfigMap.
+
+### Function invocation
+
+Get the package:
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/enforce-gatekeeper/imperative .
+```
+
+Create a directory for storing the structured output.
+
+```shell
+$ cd imperative
+$ mkdir results
+```
+
+Run the function:
+
+```shell
+kpt fn eval --image=gcr.io/kpt-fn/enforce-gatekeeper:unstable --results-dir=results
+```
+
+### Expected result
+
+You should see the output like the following:
+
+```
+[RUNNING] "gcr.io/kpt-fn/enforce-gatekeeper:unstable"
```

Comment:
You may not want to show the non-structured output, it's likely going to change, and these dos will become stale.